### ISSUE TITLE
Slow start if only process

### ISSUE THREAD

**Issue by Miosame** [gnunn1/tilix#1994]
OS: Linux Mint 20
I've noticed tilix takes ~5 seconds to start if there's no other tilix window open, is there a workaround/fix for that?

**Comment by Syphdias** [comment]
I have a similar problem which occurs after a reboot. I wonder if it is related
The first launch takes a while, but if you close tilix and open it again, it starts just fine.

I think in my case it comes down to dbus not being ready yet, but I am unsure.

Could you test if this is the same behavior for you?

**Comment by Miosame** [comment]
I've since switched to arch, where the problem doesn't seem to appear, so most likely it has to do with _some_ package being outdated or different - which one - I don't know.

**Comment by Syphdias** [comment]
Okay, thanks! Unfortunately not my issue then. I am on an Arch-ish (Manjaro) and it's a recent problem. :shrug:

**Comment by baishi** [comment]
This is to confirm after replacing xdg-desktop-portal-gtk with xdg-desktop-portal-kde the problem is solved for me.

**Comment by Syphdias** [comment]
I upgraded to `xdg-desktop-portal-gtk` from version 1.10.0-3 to version 1.12.0-1 yesterday and after that, startup time seems fine.
I checked the release notes: https://github.com/flatpak/xdg-desktop-portal-gtk/releases/tag/1.12.0 and they say
> Enable settings and appchooser portals by default

There seems to be an issue with "Slow launch of GTK3 apps with dbus-run-session" (https://github.com/flatpak/xdg-desktop-portal-gtk/issues/332) which is mentioned in a recently merged PR (https://github.com/flatpak/xdg-desktop-portal-gtk/pull/362). The PR is about something else but I wonder if it fixed this issue by accident.

I updated a bunch of packages though, so it might be something else that fixed it. And this is a total coincidence.

TL;DR: If you use `xdg-desktop-portal-gtk` < 1.12.0, try updating first.
